ui-lib/src/androidTest/java/co/electriccoin/zcash/ui/screen/backup/view/BackupTestSetup.kt

```diff
@@ -50,9 +49,9 @@ class BackupTestSetup(
         return state.current.value
     }
 
-    @SuppressLint("ComposableNaming")
     @Composable
-    fun getDefaultContent() {
+    @Suppress("TestFunctionName")
```

Comment:
I think you're right, that being under a test project is why.  The source set is src/main instead of src/androidTest, so I wonder if that's where the Studio bug might come from?

Reply:
Yeah probably. But I didn't find anything about it. So maybe we can file it to the Issue tracker and we'll see soon, hopefully.